src/explorer/CosmosDBConnection.ts

```diff
@@ -18,7 +18,12 @@ export class CosmosDBConnection extends AzureTreeItem<ISiteTreeRoot> {
 
     constructor(parent: CosmosDBTreeItem, readonly cosmosDBDatabase: CosmosDBDatabase, readonly appSettingName: string) {
         super(parent);
-        this.label = `${cosmosDBDatabase.accountName}/${cosmosDBDatabase.databaseName}`;
+        this.label = CosmosDBConnection.makeLabel(cosmosDBDatabase);
+    }
+
+    // tslint:disable-next-line:function-name
```

Comment:
nit: Rather than disable the rule for this line, can you disable it for this repo? Just change tslint.json to have the same "function-name" stuff as this: 
https://github.com/Microsoft/vscode-azureeventgrid/blob/master/tslint.json#L20